validator/client/runner.go

```diff
@@ -55,13 +55,20 @@ func run(ctx context.Context, v iface.Validator) {
 	}
 	sub := km.SubscribeAccountChanges(accountsChangedChan)
 	// Set properties on the beacon node like the fee recipient for validators that are being used & active.
-	if err := v.PushProposerSettings(ctx, km); err != nil {
-		if errors.Is(err, ErrBuilderValidatorRegistration) {
-			log.WithError(err).Warn("Push proposer settings error")
-		} else {
-			log.WithError(err).Fatal("Failed to update proposer settings") // allow fatal. skipcq
+	if v.HasProposerSettings() {
+		log.Infoln("Provided proposer settings will periodically update settings such as fee recipient in the beacon node and custom builder if enabled")
```

Comment:
What do you mean by `if enabled`? Doesn't `v.HasProposerSettings` mean that they're enabled?

Reply:
the enable is for the --enable-builder flag that is used for mev.. is there somethign I can do to make this more clear? without just saying something about mev boost

Reply:
Maybe `and custom builder (if builder is enabled)`?